core/src/main/resources/reference.conf

```diff
@@ -550,6 +550,21 @@ datastax-java-driver {
     token-map.enabled = true
   }
 
+  metrics {
```

Comment:
I think it's cool that you can enable/disable specific metrics.  If you have a metrics db (graphite, influx, etc.) that a lot of clients are feeding into, it's pretty easy to overwhelm it, so to be able to be selective about what you record is nice.